Library/Homebrew/formula_installer.rb

```diff
@@ -1232,6 +1232,11 @@ def pour
     with_env(HOMEBREW_RELOCATE_RPATHS: "1") do
       keg.replace_placeholders_with_locations tab.changed_files, skip_linkage: skip_linkage
     end
+
+    bottle = formula.bottle_for_tag(Utils::Bottles.tag.to_sym)
+    relocatable_cellar = bottle.cellar == :any || bottle.cellar == :any_skip_relocation
+    skip_prefix_relocation = relocatable_cellar || Homebrew.default_prefix?
```

Comment:
Is this equivalent to `formula.bottle_specification.skip_relocation?` at all?

I think this logic should be moved into `bottle_specification` somewhere, feels a bit brittle having it here.

Reply:
Agreed.  I wasn't as familiar with that code but it makes sense to put it there instead.

Reply:
This logic has been moved to `bottle_specification`.